tutorial.md

```diff
@@ -78,12 +78,14 @@ mkdir fn.example.io
 cd fn.example.io
 ```
 
-Now, with one command we will create the project files, build a container, and
+Now, we will create the project files, build a container, and
 deploy the function as a Knative service.
 
 
 ```bash
 faas create -l node
+faas build
+faas deploy
```

Comment:
I am not sure if this is still correct. I expect to be testing this shortly, but perhaps a reviewer can chime in to expedite the process.

Reply:
This is correct, yes.